controllers/bpfman-agent/cl_tc_program.go

```diff
@@ -199,7 +195,7 @@ func (r *ClTcProgramReconciler) findLink(attachInfoState bpfmaniov1alpha1.ClTcAt
 		// same: InterfaceName, ContainerPid, Priority, and ProceedOn.
```

Comment:
```suggestion
		// same: InterfaceName, NetnsPath, Priority, and ProceedOn.
```
Probably search for ContainerPid and see if left in any comments, just as cleanup.

Reply:
Done.